web/src/components/Plugin/PluginPage.tsx

```diff
@@ -277,12 +277,12 @@ const PluginPage: React.FC<Props> = ({
         setName(NEVER_EXIST_PLUGIN_FLAG);
       }}
       onChange={({ monacoData, formData, shouldDelete }) => {
-        let newPlugins = {
+        const newPlugins = {
           ...initialData,
           [name]: { ...monacoData, disable: !formData.disable },
         };
         if (shouldDelete === true) {
-          newPlugins = omit(newPlugins, name);
+          newPlugins[name] = null;
```

Comment:
This method is not very good, we do not usually modify the value of an object's properties directly.
An extended question, we should normally have an indication of successful deletion when we delete data.

Reply:
OK ,I will modify it again.